### PR TITLE
AlisonZ set up baseline structure 

### PR DESCRIPTION
**Pull request by AlisonZ**
# API Muncher
Congratulations! You're submitting your assignment!

## Comprehension Questions
| Question | Answer    |
| :------------- | :------------- |
| How did you go about exploring the Edamam API, how did you try querying the API? |  |
| Describe your API Wrapper.  How did you decide on the methods you created? |  |
| Describe an edge case or failure case test you wrote for your API Wrapper. |  |
| Explain how VCR aids in testing an API. |  |
| What is the Heroku URL of your deployed application? |  |
| Do you have any recommendations on how we could improve this project for the next cohort? |  |

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# See https://help.github.com/articles/ignoring-files for more about ignoring files.
```
Comment (CheezItMan) [inline .gitignore:1]: Remembrer to add `.env` to your `.gitignore` file.

**Review by CheezItMan — CHANGES_REQUESTED** [review]
Just one change hiding your API Key from Github.

**Review by CheezItMan — APPROVED** [review]
Great work
